src/modules/ui.js

```diff
@@ -0,0 +1,31 @@
+import { todos, renderTodoList, todoList } from './todos.js';
+// eslint-disable-next-line import/no-unresolved
+import { addItem } from './ui-actions.js';
```

Comment:
Your app throws the error below and refuses to run because you are importing `ui-actions.js` whereas you have `ui-action.js`, and also after resolving these another error will greet you and its simply because you are importing a default export as a named export, instead of `import { addItem } from './ui-action.js';`, you should have `import addItem from './ui-actions.js';`, after which you can remove the eslint disable, consider doing so in order for the app to run successfully.

<img width="960" alt="image" src="https://github.com/Edgahkipkemoi/To-do-list/assets/84930355/cf1a95e6-94d3-4faa-a504-c287fe7c7f19">


NOTE: ensure all functionalities are intact before asking for review